deltachat-rpc-server/src/main.rs

```diff
@@ -67,10 +68,22 @@ async fn main_impl() -> Result<()> {
     // Logs from `log` crate and traces from `tracing` crate
     // are configurable with `RUST_LOG` environment variable
     // and go to stderr to avoid interferring with JSON-RPC using stdout.
-    tracing_subscriber::fmt()
-        .with_env_filter(EnvFilter::from_default_env())
-        .with_writer(std::io::stderr)
-        .init();
+    tracing::subscriber::set_global_default({
```

Comment:
I think `#[allow(unexpected_cfgs)]` should be moved here, as in `deltachat-repl`